int64 client_time_req
---
int64 server_time_req
int64 server_time_resp
